<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WRobotPageClass</class>
 <widget class="QWidget" name="WRobotPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>598</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>122323</width>
    <height>232323</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WRobotPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 12px &quot;微软雅黑&quot;;
}

QFrame#frame_chat[is_sel=true]{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0,102,255,25), stop:1 rgba(189,0,255,25));
	border-radius:8px;
	border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);
}

QFrame#frame_chat[is_sel=false]{
	border:1px solid #000000;
	border-radius:8px;
	background-color: #ffffff;
}

QFrame#frame_sound[is_sel=false]{
	border:1px solid #000000;
	border-radius:8px;
	background-color: #ffffff;
}

QFrame#frame_sound[is_sel=true]{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0,102,255,25), stop:1 rgba(189,0,255,25));
border-radius:8px;
border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);
}

QLabel#lb_robot_icon{
	background-image: url(:/QtTest/icon/Setting/robot_icon.png);
	background-position: center;
    background-repeat: no-repeat;
	background-color: transparent;
}

QLabel#lb_chat,#lb_sound_name{
	font: 18px &quot;微软雅黑&quot;;
	background-color: transparent;
}

QPushButton{
	border:none;
	background-position: center;
    background-repeat: no-repeat;
	background-color: transparent;
}

QPushButton#pb_robot_setting{
	background-color: transparent;
	background-position: left;
    background-repeat: no-repeat;
	background-image: url(:/QtTest/icon/Setting/set.png);
}


QLabel#lb_sound{
	background-image: url(:/QtTest/icon/Setting/sound_icon.png);
	border:none;
	background-position: center;
    background-repeat: no-repeat;
	background-color: transparent;
}

QPushButton#pb_sound_set{
	background-color: transparent;
	background-position: left;
    background-repeat: no-repeat;
	background-image: url(:/QtTest/icon/Setting/set.png);
}

QCheckBox{
	font: 20px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

QCheckBox::indicator {width: 30px; height: 30px;}

QCheckBox::indicator:unchecked {image: url(:/QtTest/icon/Setting/off.png);}

QCheckBox::indicator:checked {image: url(:/QtTest/icon/Setting/open.png);}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_chat">
     <property name="minimumSize">
      <size>
       <width>637</width>
       <height>220</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>647</width>
       <height>220</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>16</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lb_robot_icon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lb_chat">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string>ChatBot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lb_aced">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string>(Activated)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lb_robot_name">
          <property name="minimumSize">
           <size>
            <width>482</width>
            <height>90</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>xxxxxxxxxxx</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_robot_setting">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>14</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);</string>
            </property>
            <property name="text">
             <string>Setup</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>128</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="pb_robot_lock">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>58</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>58</width>
              <height>19</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string>Activate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_sound">
     <property name="minimumSize">
      <size>
       <width>637</width>
       <height>220</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>647</width>
       <height>220</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="topMargin">
         <number>16</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="lb_sound">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>41</width>
              <height>38</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lb_sound_name">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string>Sound Bot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lb_sound_act">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string>(Activated)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lb_sound_tip">
          <property name="minimumSize">
           <size>
            <width>482</width>
            <height>90</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>xxxxxxxxxxxxxxxxx</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_sound_set">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>14</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);</string>
            </property>
            <property name="text">
             <string>Setup</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="pb_sound_lock">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string>Activate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="minimumSize">
             <size>
              <width>58</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>58</width>
              <height>19</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255,0);</string>
            </property>
            <property name="text">
             <string>Activate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>105</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../AiSound.qrc"/>
 </resources>
 <connections/>
</ui>
